news/4946.feature

```diff
@@ -0,0 +1 @@
+Use all the apiExpanders in use, so we perform a single request for getting all the required data. @sneridagh
```

Comment:
I am no Steve, but I will still suggest a minor change just to avoid the repetition of the word "use" and to clarify 😛 

```suggestion
Use all the available apiExpanders, so we perform a single request for getting all the required data. @sneridagh
```